locales/en/messages.json

```diff
@@ -3663,7 +3663,7 @@
         "message": "Dynamic Idle Value [rpm]"
     },
     "pidTuningIdleMinRpmHelp": {
-        "message": "Set this parameter to provide an rpm based floor-value as a safety net against motor desync.<br>A racer might prefer very good response and low rates - therefore a low Dynamic Idle[rpm] and a high Dshot Idle value may be useful. Freestylers and LOS pilots will appreciate very low thrust at idle and might want to use less Dshot Idle value while using a bit more Dynamic Idle[rpm] to avoid desyncs at high rates. Note that the Dynamic Idle[rpm] value always must be less than Dshot Idle value.<br><br>Visit this <a href=\"https://github.com/betaflight/betaflight/wiki/Tuning-Dynamic-Idle\"target=\"_blank\" rel=\"noopener noreferrer\">wiki</a> entry for more info."
+        "message": "Set this parameter to dynamically adjust the minimum DShot value based on the rpm and avoid motor desyncs.<br>Lower Dynamic Idle [rpm] with higher DShot Idle provides more response at low throttle, better for racing.<br>Higher Dynamic Idle [rpm] with lower DShot Idle provides less unwanted thrust at low throttle, better for freestyle.<br>Note that the Dynamic Idle [rpm] value must always be less than the DShot Idle value.<br><br>Visit this <a href="https://github.com/betaflight/betaflight/wiki/Tuning-Dynamic-Idle"target="_blank" rel="noopener noreferrer">wiki</a> entry for more info."
```

Comment:
The backslashes in the string need to be escaped: `\"`

Reply:
Done. how the hell did the Travis CI build succeeded with this major flaw?! JS sure is forgiving

Reply:
@leocb: It's not about being 'forgiving', it's about being lazy and only evaluating most things at runtime (and then happily crashing).
But you are making a good point - it might make sense to add a test for this. And I am halfway surprised that SonarCloud does not do a correctness check for JSON.